src/EditorFeatures/CSharpTest/AssignOutParameters/AssignOutParametersAboveReturnTests.cs

```diff
@@ -143,30 +140,31 @@ bool Try(out int i)
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAssignOutParameters)]
         public async Task TestMissingWhenAssignedThroughAllPaths()
         {
-            await TestMissingInRegularAndScriptAsync(
-@"class C
+            var code = @"class C
 {
-    char M(out int i)
+    char M(bool b, out int i)
     {
         if (b)
             i = 1;
         else
             i = 2;
         
-        [|return 'a';|]
+        return 'a';
     }
-}");
+}";
+
+            await VerifyCS.VerifyCodeFixAsync(code, code);
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAssignOutParameters)]
         public async Task TestMultiple()
         {
-            await TestInRegularAndScriptAsync(
+            await VerifyCS.VerifyCodeFixAsync(
 @"class C
 {
     char M(out int i, out string s)
     {
-        [|return 'a';|]
+        {|CS0177:{|CS0177:return 'a';|}|}
```

Comment:
Interesting, duplicate diagnostics need nested annotation?

Reply:
Interested in hearing what this is about as well.

Reply:
All diagnostics must be declared. There is no such thing as a "nested" annotation; this is just a visual artifact caused by one diagnostic's span being a superset of another.

Reply:
>  one diagnostic's span being a superset of another.

but in this case, the spans are identical.  

1. does this indicate an issue with the analyzer (producing duplicate diagnostics)
2. if they're identical, does the test need to declare both?

Reply:
There is no analyzer here. The compiler reported both diagnostics.